src/iteration.jl

```diff
@@ -195,15 +195,19 @@ end
 Base.propertynames(lt::LazyTree) = propertynames(innertable(lt))
 Base.getproperty(lt::LazyTree, s::Symbol) = getproperty(innertable(lt), s)
 
-# a specific branch
+Base.broadcastable(lt::LazyTree) = lt
+Base.IndexStyle(::Type{<:LazyTree}) = IndexLinear()
+Base.findall(f::Function, lt::LazyTree) = [f(evt) for evt in lt]
```

Comment:
Can this be `f.(lt)`? Then it returns a more compact BitVector instead of Vector{Bool}

Reply:
Ah wait, is this the same behavior as `findall([true, false, true, false])` which gives `[1,3]`?

Reply:
wait, you're right, this is not normal `findall`

Reply:
Base has
```julia
findall(testf::Function, A) = collect(first(p) for p in pairs(A) if testf(last(p)))

# Broadcasting is much faster for small testf, and computing
# integer indices from logical index using findall has a negligible cost
findall(testf::Function, A::AbstractArray) = findall(testf.(A))
```

Reply:
If normal broadcasting works now, then the Base implementation should just work